app/scripts/controllers/eventOverview.js

```diff
@@ -20,12 +20,23 @@ angular
       footer: true,
     };
 
+    $scope.resetImage = () => {
+      $scope.imageSrc = conference.image;
+      $scope.includeImageToAllPages = conference.includeImageToAllPages;
+    };
+
     $scope.conference = conference;
-    $scope.imageSrc = '';
+    $scope.imageSrc = conference.image;
```

Comment:
I assume when an event is created, the image is defaulted to `''` right?

Reply:
That is true, if there is already an imaf]ge for the conference then it should be displayed on overview page